compiler/llvm/clangUtil.cpp

```diff
@@ -3830,9 +3831,48 @@ void addDumpIrPass(const PassManagerBuilder &Builder,
   PM.add(createDumpIrPass(llvmPrintIrStageNum));
 }
 
-static void linkLibDevice() {
-  // We follow the directions in https://llvm.org/docs/NVPTXUsage.html#libdevice
+static void linkBitCodeFile(const char *bitCodeFilePath) {
+  GenInfo* info = gGenInfo;
+
+  // load into new module
+  llvm::SMDiagnostic err;
+  auto libdevice = llvm::parseIRFile(bitCodeFilePath, err,
```

Comment:
Maybe change the variable name here to something more generic? `lib`, `bcLib`...